dir.props

```diff
@@ -71,6 +72,10 @@
   </PropertyGroup>
   <Import Project="$(ToolRuntimePath)/net45/roslyn/build/Microsoft.Net.Compilers.props" Condition="'$(UseRoslynCompilers)'!='false'" />
 
+  <PropertyGroup>
+    <_BuildAgainstPackages Condition="'$(KeepAllProjectReferences)' != 'true' and $([System.Text.RegularExpressions.Regex]::IsMatch('$(MSBuildProjectFullPath)', '\\tests\\', System.Text.RegularExpressions.RegexOptions.IgnoreCase))">$(BuildTestsAgainstPackages)</_BuildAgainstPackages>
```

Comment:
The intention is to utilize this property as the pivot which determines whether we import buildagainstpackages.targets or not because if we import that targets file then we change some of the build order targets and such.  BuildTools:Build.common.props seems like a good place for this though.